app/workers/publish_worker.rb

```diff
@@ -30,6 +30,6 @@ def print_limitations_of_publishing(document)
     message = "Skipped publishing document with content_id #{content_id}"
     message += " because it has a state of '#{state}'."
 
-    puts message
+    Rails.logger.warn message
```

Comment:
Sidekiq has its own logger available in worker objects so we don't need the `Rails` prefix here. See https://github.com/alphagov/specialist-publisher/blob/master/app/workers/email_alert_api_worker.rb#L9